policies/tiles.md

```diff
@@ -5,75 +5,188 @@ permalink: /policies/tiles/
 
 # {{ page.title }}
 
-We are in principle happy for our map tiles to be used by external users for creative and unexpected uses – in contrast to most web mapping providers, which insist that you use only their supplied API.
+OpenStreetMap (OSM) data is **free for everyone to use**. Our **tile servers are not**: they are [funded by donations](https://supporting.openstreetmap.org/donate/) and [sponsorship](https://hardware.openstreetmap.org/thanks/), and capacity is limited. To protect the service for mappers and the wider community, you **must** follow this policy when using tiles from `tile.openstreetmap.org`.
 
-However, **OpenStreetMap's own servers are run entirely on donated resources.** They have strictly limited capacity. Heavy use of OSM tiles adversely affects people's ability to edit the map, and is an abuse of the individual donations and sponsorship which provide hardware and bandwidth. As a result, we require that users of the tiles abide by this tile usage policy.
+We welcome creative uses and do not require you to use a specific API. However, heavy or inappropriate use harms others' ability to edit and view the map. We may block access, without notice, if your usage degrades the service. We may try to contact you if possible, but cannot guarantee this.
 
-**OpenStreetMap data is [free for everyone to use](https://wiki.openstreetmap.org/wiki/OpenStreetMap_License). Our tile servers are not.**
+Availability is **best-effort**: there is **no SLA or guarantee**. All [OSMF](https://osmfoundation.org/) services are also covered by the [OSMF Terms of Use](https://wiki.osmfoundation.org/wiki/Terms_of_Use).
 
-Below are the minimum requirements that users of tile.openstreetmap.org must adhere to. These may change in future, depending on available resources. Should any users or patterns of usage nevertheless cause problems to the service, access may still be blocked without prior notice. We will try to contact relevant parties if possible, but cannot guarantee this.
+If you cannot meet these requirements, please use an **alternative OSM-derived service** (see below), or run your own.
 
-Although our map tiles are generally very reliable, their availability to others is on a best effort basis and we offer no SLA or guarantees.
+---
```

Comment:
I don't think we should add line breaks between each section.

Reply:
Removed from current draft.